### PR TITLE
Updates to Documentation

### PR DESCRIPTION
**Pull request by CleanMachine1**
Hi, I have fixed some errors with the documentation and added a few bits.
I love to have clean documentation because I hate when software isn't properly documented.
I plan on fixing more, so leave this PR open

Hope you appreciate what I have done, even though its not a lot (this will change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-stacer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/stacer-rpi
+Stacer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/stacer-rpi
 
-teamviewer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/teamviewer
+Teamviewer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/teamviewer
 
-vscodium https://github.com/chunky-milk/pi-bashscripts-files/tree/main/vscodium
+VSCodium https://github.com/chunky-milk/pi-bashscripts-files/tree/main/vscodium
 
-## other useless stuff
```
Comment (CleanMachine1) [inline README.md:54]: This made me laugh, I hope its an error

**Comment by ryanfortner** [comment]
> Hi, I have fixed some errors with the documentation and added a few bits.
> I love to have clean documentation because I hate when software isn't properly documented.
> I plan on fixing more, so leave this PR open
> 
> Hope you appreciate what I have done, even though its not a lot (this will change)

Thanks for this. I've been busy with [another project](https://github.com/chunky-milk/raspbian-addons), so I haven't been able to keep up with this repo as much.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,59 @@
...
-dolphin emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
+Dolphin Emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
 
-dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
+Dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
 
 Duckstation https://github.com/chunky-milk/pi-bashscripts-files/tree/main/duckstation
 
-freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
+Freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
```
Comment (Itai-Nelken) [inline README.md]: FreeTube is already on pi-apps, so this line can be removed.

File: README.md [hunk 3]
```diff
@@ -1,59 +1,59 @@
...
+Google Earth https://github.com/chunky-milk/pi-bashscripts-files/tree/main/google-earth
 
-lutris game manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/lutris
+Lutris game manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/lutris
 
-megasync https://github.com/chunky-milk/pi-bashscripts-files/tree/main/megasync
+Megasync https://github.com/chunky-milk/pi-bashscripts-files/tree/main/megasync
 
-obs studio https://github.com/chunky-milk/pi-bashscripts-files/tree/main/obs-studio
+OBS Studio https://github.com/chunky-milk/pi-bashscripts-files/tree/main/obs-studio
```
Comment (Itai-Nelken) [inline README.md]: OBS Studio is also already on pi-apps, so the line can be removed.

File: README.md [hunk 4]
```diff
@@ -1,59 +1,59 @@
...
 
-openscad https://github.com/chunky-milk/pi-bashscripts-files/tree/main/openscad
+Openscad https://github.com/chunky-milk/pi-bashscripts-files/tree/main/openscad
```
Comment (Itai-Nelken) [inline README.md]: there is a issue for OpensCad in the pi-apps repo already (https://github.com/Botspot/pi-apps/issues/556).

File: README.md [hunk 5]
```diff
@@ -1,59 +1,59 @@
...
+Remmina https://github.com/chunky-milk/pi-bashscripts-files/tree/main/remmina
 
-stacer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/stacer-rpi
+Stacer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/stacer-rpi
 
-teamviewer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/teamviewer
+Teamviewer https://github.com/chunky-milk/pi-bashscripts-files/tree/main/teamviewer
 
-vscodium https://github.com/chunky-milk/pi-bashscripts-files/tree/main/vscodium
+VSCodium https://github.com/chunky-milk/pi-bashscripts-files/tree/main/vscodium
```
Comment (Itai-Nelken) [inline README.md]: VSCodium is in pi-apps as well.

**Comment by ryanfortner** [comment]
Let me know when to merge.

File: README.md (hunk 2 above)
Comment (CleanMachine1) [reply README.md]: Why are the folder still there, should I remove them?

**Comment by CleanMachine1** [comment]
@chunky-milk I understand that this is more of a personal project, but could you share a screenshot of the traffic?
I would love to see how many are actually using this amount of scripts

**Comment by ryanfortner** [comment]
image
image2

**Comment by CleanMachine1** [comment]
Thats a shame, it seems to be dead :(

**Comment by ryanfortner** [comment]
@CleanMachine1 is it ready to merge

**Comment by CleanMachine1** [comment]
Nope, going to need a few days, got to fix every file capitals and other things, can't do half a job

If you would like to help I would appreciate it a lot.

**Comment by CleanMachine1** [comment]
@chunky-milk after I am done with this would you like for me to work on the other repo you maintain, the addon one

**Comment by ryanfortner** [comment]
> @chunky-milk after I am done with this would you like for me to work on the other repo you maintain, the addon one

Maybe. ;)

**Comment by CleanMachine1** [comment]
Wow thats a lot of commits

**Comment by Itai-Nelken** [comment]
nice!
there is a way newer release of Github-cli (`gh`): https://github.com/cli/cli/releases/tag/v1.8.1

**Comment by CleanMachine1** [comment]
Ok, I wasn't checking any file names.
To be honest, I feel I am wasting my time a little because no one is using it and my changes are mostly pointless

If you would like to push this PR, go for it, if not I'll finish soon

**Comment by CleanMachine1** [comment]
@Itai-Nelken I'll add that in my next commit

**Comment by CleanMachine1** [comment]
@chunky-milk Feel free to push, I am done

**Pull request merged**